scripts/update_deps.sh

```diff
@@ -0,0 +1,25 @@
+PYTHON_COMMAND=python3
+PIP_COMMAND="$PYTHON_COMMAND -m pip"
+
+info() { echo -e "\033[33mINFO: $1\033[0m\n"; }
+error() { echo -e "\033[31mERROR: $1\033[0m" && exit 1; }
+
+# Cleanup python build directory
+rm -rf /deps/build/*
+
+$PIP_COMMAND install --progress-bar=off --no-deps --exists-action=w -r "requirements/pip.txt"
+
+info "Installing prod dependencies"
+$PIP_COMMAND install --progress-bar=off --no-deps --exists-action=w -r "requirements/prod.txt"
+
+# Install dev dependencies unless --prod is passed
+if [[ ! "$@" == *"--prod"* ]]; then
```

Comment:
@eviljeff @willdurand how would I do this in the make file? Just curious as it wasn't obvious that I could add conditions like this at least not in a way that didn't seem much simpler in a bash script.

Reply:
You can evaluate args in Makefiles as well but that is generally not the recommended way of doing this. If this was still in a `Makefile` the easiest way to do it IMHO would be to have different targets, with the "prod" target depending on the dev one, which is more or less what we had before.

Reply:
The only way to make this really work the way we want was to extract a `update_deps_base` target. This allows us to install the pip and node modules, install olympia, clean the build dir etc.

Then we have the original `update_deps` which acts as a "dev" dependencies which installs python dev deps and finally a `update_deps_prod` which prunes the node modules dev dependencies.